src/domain/Tables/Table/subcomponents/TableRow.tsx

```diff
@@ -180,81 +168,40 @@ const getDataCells = <T extends {}>(
         )
       }
 
-      const cellContentOverride = (contentOverride && Array.isArray(contentOverride(data))) ? contentOverride(data) as JSX.Element[] : undefined
-
       return (
         <TableRowDataCell<T>
-          key={`${column.name}-${key}`}
+          key={`${column.name}-${id}`}
```

Comment:
I removed `key` and the content override code. A much more elegant solution to this is to just make SPA change the id of the element when you want a transition animation. I made it just append the upload status to the id of the row which seems to make it work and means we don't need to try magicking it in ui components.